sgx/pelz_enclave.edl

```diff
@@ -44,8 +45,44 @@ enclave {
  *
  * @return 0 on success, 1 on error
  */
-	public int key_table_delete(charbuf key_id);
+public int key_table_delete(charbuf key_id);
 
+/**
+ * <pre>
+ * This function initializes a server table for lookup.
+ * <pre>
+ *
+ * @param[in] server_table The pointer for server table to be initialized
+ * @param[out] server_table The initialized server table
+ *
+ * @return 0 on success, 1 on error
+ */
+public int server_table_init(void);
+
+/**
+ * <pre>
+ * This function destroys the server table.
+ * <pre>
+ *
+ * @param[in] server_table The server table to be destroyed
+ *
+ * @return 0 on success, 1 on error
+ */
+public int server_table_destroy(void);
+
+/**
+ * <pre>
+ * This function to delete values in hash table based on location in key_id.
+ * <pre>
+ *
+ * @param[in] server_id.chars Server identifier assumed to be null terminated
+ * @param[in] server_id.len The length of the server identifier
+ * @param[in] server_table The server table that the cert needs to be deleted from
```

Comment:
This function doesn't take the ```server_table``` as an argument.

Reply:
```server_table``` is an argument for the function and is used in the function. The function intakes ```server_table``` as an extern variable so you don't have to call the variable as an input argument which is the same way we do it for the key table.